cep/html/panel.html

```diff
@@ -90,7 +90,14 @@
       // Should run when panel start is requested
       window.addEventListener("load", sg_panel.Panel.on_load);
       // Should run when panel is closed down.
-      window.addEventListener("unload", sg_panel.Panel.on_unload);
+      window.addEventListener("unload", function() {
+        try {
+          var event = new CSEvent("com.adobe.csxs.events.CloseManager", "APPLICATION");
+          sg_panel.Panel._cs_interface.dispatchEvent(event);
+        } catch (error) {
+          console.error("Error closing extension:", error);
+        }
```

Comment:
Is it possible to add this logic to `sg_panel.Panel.on_unload` function?

Reply:
That was my initial approach, but it's interesting because nothing inside .on_unload() is being executed. In fact, there's only a log there, which doesn’t do anything. I also tried on_unload.bind(), but that didn’t work either.

Reply:
done